### PR TITLE
[EXAMPLE] Popover - static

### PR DESCRIPTION
**Pull request by aibcmars**
Add custom popover for the task found by it's id

https://cdn.statically.io/gh/process-analytics/bpmn-visualization-examples/execution-data_popover_info/examples/index.html

### CODE REVIEW

File: examples/custom-navigation/popover/index.js
Comment (tbouffard) [inline examples/custom-navigation/popover/index.js:2]: :warning: should use #111

**Comment by csouchet** [comment]
## MacOs
### Firefox
_**No Pop Over 🙁**_ 
image

### Chrome
image

### Safari
_**No Pop Over 🙁**_ 
image

File: examples/custom-navigation/popover/index.js
Comment (csouchet) [inline examples/custom-navigation/popover/index.js:184]: ❓ Can we put the method for the button behaviour in another script, and keep here only the code related to the popover configuration ?

**Comment by csouchet** [comment]
I tought a popover is displayed only when the mouse is over the HTML element with the listener, like: https://picturepan2.github.io/spectre/components/popovers.html

File: examples/custom-navigation/popover/index.js
Comment (aibcmars) [reply examples/custom-navigation/popover/index.js:2]: waiting for merge of #111

File: examples/custom-navigation/popover/README.md
Comment (aibcmars) [reply examples/custom-navigation/popover/README.md:17]: sure

File: examples/custom-navigation/popover/README.md
Comment (aibcmars) [reply examples/custom-navigation/popover/README.md:1]: ok

File: examples/custom-navigation/popover/index.html
Comment (aibcmars) [reply examples/custom-navigation/popover/index.html]: Yes, actually you are right,
Adding this differentiation can add unnecessary complexity.

I am OK to remove it - let me know if you want to discuss anything in particular or I can proceed with removal

**Comment by aibcmars** [comment]
@csouchet 
I have changed the sequence of disappearing / appearing elements for popover
it helped for Firefox display on Windows.

please confirm it is OK now for Mac:
- [ ] Firefox
- [ ] Safari

**Comment by tbouffard** [comment]
:heavy_check_mark: display ok on Ubuntu 18.04, FF 83.0 and Chromium 87.0.4280.88 for commit 131456c

**Comment by csouchet** [comment]
The div appears well in Firefox & Chrome, not Safari, but it appears all time even if I don't move my mouse on the task. 
For me, I don't consider it as a popover.

File: examples/custom-navigation/popover/index.js
Comment (tbouffard) [reply examples/custom-navigation/popover/index.js]: rebased on master after #11 merge

**Review by tbouffard — APPROVED** [review]

**Pull request merged**
